superduperdb/ext/llm/base.py

```diff
@@ -0,0 +1,317 @@
+import abc
+import asyncio
+import dataclasses as dc
+import inspect
+import typing
+from functools import reduce, wraps
+from logging import WARNING, getLogger
+from typing import Any, Callable, List, Optional, Union
+
+from superduperdb import logging
+from superduperdb.components.component import Component
+from superduperdb.components.model import _Predictor
+from superduperdb.ext.utils import format_prompt
+
+if typing.TYPE_CHECKING:
+    from superduperdb.base.datalayer import Datalayer
+
+# Disable httpx info level logging
+getLogger("httpx").setLevel(WARNING)
```

Comment:
Please explain.

Reply:
The OpenAI package will print the INFO information of network-related requests when calling the interface, which is of little significance. It is disabled here.

Reply:
Ok